contrib/Tests/Unit/ZipkinExporterTest.php

```diff
@@ -2,10 +2,10 @@
 
 declare(strict_types=1);
 
-namespace OpenTelemetry\Tests\Unit\Exporter;
+namespace OpenTelemetry\Contrig\Tests\Unit;
```

Comment:
I think this should be `Contrib`, not `Contrig`

Reply:
fixed